packages/mui-material-pigment-css/src/Grid/MaterialPigmentGrid.ts

```diff
@@ -0,0 +1,3 @@
+import Grid from '@pigment-css/react/Grid';
```

Comment:
The plan is to eventually move the implementation here, right? (I'm ok with doing it later)

Reply:
Yes. That's the idea. Since stable release of Pigment won't have these.